network/README.md

```diff
@@ -20,10 +20,7 @@ The network component uses:
 * TCP for reliable transport.
 * [Noise](https://noiseprotocol.org/noise.html) for authentication and full
  end-to-end encryption.
-* [Yamux](https://github.com/hashicorp/yamux/blob/master/spec.md) for
-multiplexing substreams over a single connection.
-* Push-style [gossip](https://en.wikipedia.org/wiki/Gossip_protocol) for peer
-discovery.
+* On-chain addresses for discovery, with a seed peers file for initial discovery.
```

Comment:
this PR is already improving the README but additional stuff could be link to specs folder, mention that we split network messages per internal components, that we have a component negotiation, etc.

Reply:
That's a good point, this is what @JoshLind did: https://github.com/libra/libra/tree/master/secure/key-manager/

Reply:
Spec added